tests/unit_test/channels/mail_scheduler_test.py

```
@@ -0,0 +1,84 @@
+        yield {
+            'mock_client': mock_client_instance,
+            'mock_channels': mock_channels,
+            'mock_process_mails': mock_process_mails,
+            'mock_jobstore_init': mock_jobstore_init,
+        }
+
+
+@pytest.mark.asyncio
+async def test_mail_scheduler_epoch_creates_scheduler(setup_environment2):
+    with patch("apscheduler.schedulers.background.BackgroundScheduler.start", autospec=True) as mock_start, \
+            patch("apscheduler.schedulers.background.BackgroundScheduler.add_job", autospec=True) as mock_add_job:
```

Ruff

Local variable `mock_add_job` is assigned to but never used

Remove assignment to unused variable `mock_add_job`

(F841)